pkgs/top-level/cuda-packages.nix

```diff
@@ -50,10 +49,14 @@ let
     cudaAtLeast = strings.versionAtLeast cudaVersion;
 
     # Maintain a reference to the final cudaPackages.
-    # Without this, if we use `final.callPackage` and a package accepts `cudaPackages` as an argument,
-    # it's provided with `cudaPackages` from the top-level scope, which is not what we want. We want to
-    # provide the `cudaPackages` from the final scope -- that is, the *current* scope.
-    cudaPackages = final;
+    # Without this, if we use `final.callPackage` and a package accepts `cudaPackages` as an
+    # argument, it's provided with `cudaPackages` from the top-level scope, which is not what we
+    # want. We want to provide the `cudaPackages` from the final scope -- that is, the *current*
```

Comment:
The change is fine within the scope of the PR, but the comment is misleading because packages under `cudaPackages` should not ask for the entire scope as an argment, they should ask for individual components CC @ConnorBaker